src/state/routing/useRoutingAPITrade.ts

```diff
@@ -44,83 +46,33 @@ export function useRoutingAPITrade<TTradeType extends TradeType>(
   })
 
   const {
-    isLoading,
     isError,
-    data: quoteResult,
-    currentData,
+    data: tradeResult,
+    currentData: currentTradeResult,
```

Comment:
Question: how is `data` different from `currentData`? Could we have a comment there possibly?

Reply:
I see we're comparing them to know whether things are syncing or not. Are these coming from different APIs or sources?

Reply:
it's a thing from rtk-query - `data` is the cached result from the previous query run, and while refetching, `data` and `currentData` can be different

https://redux-toolkit.js.org/rtk-query/usage/queries#frequently-used-query-hook-return-values